### DISCUSSION TITLE
Auto archive year-month-day

### DISCUSSION THREAD

**Discussion by lucamancusodev** [Mailu/Mailu#3256] [User support] [answered]
## Environment & Version

- `docker compose v2.27.0`
- Version: `2.0`

## Description
I would like to automatically archive inbox folder in year-month-day folders so it's easier to search emails.
In webmail roundcube there is "special folders" and it works only if I click "archive", no way to automate.

## Replication Steps

Fresh install on email account
## Observed behaviour

No way to create rule like autoarchive
## Expected behaviour

Auto archive in web admin side or server side
![immagine](https://github.com/Mailu/Mailu/assets/41821623/6dc9b4f8-8c44-4c8a-b95b-bbf44146ccd6)

**Comment by nextgens** [comment]
Use managesieve to create a server-side rule (/webmail/?_task=settings&_action=plugin.managesieve)

https://thsmi.github.io/sieve-demo/libSieve/SieveGui.html can assist you with the syntax (enable all capabilities)

**Reply by lucamancusodev** [reply]
Hi,
Thank you for the hint, I tried to create a new filter script and the problem is on creating the folder if not exist.
If i manually create the folder, the filter moves email.

![immagine](https://github.com/Mailu/Mailu/assets/41821623/dc404a7b-15ab-4815-af0a-042f253e3706)

**Reply by nextgens** [reply]
You probably need another capability in your require to make it work.

See https://www.iana.org/assignments/sieve-extensions/sieve-extensions.xhtml

**Reply by nextgens** [reply, marked as answer]
You need "mailbox", see https://www.rfc-editor.org/rfc/rfc5490.html#section-3.2

**Reply by lucamancusodev** [reply]
Thank you, missed that info.
